package.json

```diff
@@ -44,7 +44,8 @@
     "eslint": "latest",
     "nuxt": "^3.0.0",
     "typescript": "^4.9.5",
-    "ufo": "^1.1.1"
+    "ufo": "^1.1.1",
+    "uuid": "^9.0.0"
```

Comment:
Why do you added the `uuid` package here? I can't see a place where you use it?

Reply:
I personally don't, but it was used in the docs and i needed to install it to run them, it can be omitted.

Reply:
Please update this in `docs/package.json`